dfdaemon/initializer/initializer.go

```diff
@@ -148,7 +146,15 @@ func initLogger() {
 		if logFile, err := os.OpenFile(logFilePath, os.O_CREATE|os.O_RDWR|os.O_APPEND, 0644); err == nil {
 			logFile.Seek(0, 2)
 			log.SetOutput(logFile)
-			go rotateLog(logFile, logFilePath)
+			go func(logFile *os.File) {
+				ticker := time.NewTicker(60 * time.Second)
+				for range ticker.C {
+					if err := rotateLog(logFile); err != nil {
+						// write error log into os.Stderr directly to avoid writing to a broken log file.
+						fmt.Fprintf(os.Stderr, "failed to rotate log %s: %v\n", logFile.Name(), err)
```

Comment:
I see that function `rotateLog` ensures that it will reset `logFile` as the output of log, so maybe we can just use `log` here?

Reply:
@lowzj 

OK, i just afraid when we failed to rotate the log, so weird thing happened on the logFile that will lead to write the log into log file failed, but i'm not sure what the weird things are.

Reply:
@lowzj 

Alternatively, what about we use MultiWriter?

```
// log both file and console
wrt := io.MultiWriter(os.Stdout, f)
// set output
log.SetOutput(wrt)
```

Reply:
@godliness I think that it's a good option for user to set when we want to debug or test locally. It's better to do this in a new pull request.